src/main/java/Application.java

```diff
@@ -0,0 +1,53 @@
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class Application {
+    public static void main(String[] args) {
+        System.out.println("경주할 자동차 이름을 입력하세요. (이름은 쉼표(,) 기준으로 구분)");
+
+        Scanner sc = new Scanner(System.in);
+        String carNames = sc.nextLine();
+
+        // 각 자동차에 이름을 부여할 수 있다.
+        // 자동차 이름은 쉼표를 기준으로 구분한다.
+        String[] carName = carNames.split(",");
+
+        try {
+
+            for (String name : carName) {
+                // 자동차 이름은 5자 이하만 가능하다.
+                if (name.length() > 5) {
+                    throw new CarNameException("자동차 이름은 5자 이하만 가능합니다.");
+                    }
+                }
+        }catch (CarNameException C){
```

Comment:
```suggestion
        } catch (CarNameException exception){
```

1. Code Style을 정해서 그에 맞게 모든 파일과 클래스를 포맷해주세요. [참고](https://sas-study.tistory.com/445)
2. 상수를 제외한 일반적인 변수명들은 소문자, camelCase를 사용해주세요.